### ISSUE TITLE
keras model save for serving 

### ISSUE THREAD

**Issue by linWujl** [keras-team/keras#10969]
Hi, i met a problem when save keras model for serving,  can i use the following method to save a model for predicting? 

        model.compile(loss='categorical_crossentropy',
                      optimizer='rmsprop',
                      metrics=['acc'])

        model.fit(x_train, y_train, validation_data=(x_val, y_val),
                  epochs=10, batch_size=256)
        print(model.input)
        print(model.output)
        signature = tf.saved_model.signature_def_utils.predict_signature_def(inputs={'text': model.input}, outputs={'features': model.output})
        builder = tf.saved_model.builder.SavedModelBuilder(os.path.join(FLAGS.buckets, 'ker1'))
        builder.add_meta_graph_and_variables(
            sess = K.get_session(),
            tags= [tf.saved_model.tag_constants.SERVING],
            signature_def_map = {'predict_article': signature}
        )
        builder.save()

**Comment by srihari-humbarwadi** [comment]
Use this 
https://github.com/srihari-humbarwadi/keras_to_tensorflow_serving/blob/master/export.py

**Comment by jalyWu** [comment]
@srihari-humbarwadi 
thanks for your answer, 
in your code, does it means that train the model first and save the model, and then load the model and transform the model for serving?
what i mean is that how to directly save the model for serving after the fit method(train procedure), 
is there any way to do that?

**Comment by srihari-humbarwadi** [comment]
@jalyWu if u want to save directly aftet training 
Call 
```K.set_learning_phase(0)```
And use the remaining part of my code 
(Jus skip the loading part)
